### ISSUE TITLE
Forms isValidForm operator fails when compute is provided

### ISSUE THREAD

**Issue by garth** [cerebral/cerebral#744]
So long as the value passed to isValidForm can be resoled to a form then it shouldn't matter where the form comes from.

The following currently throws `Cerebral Forms - isValidForm factory requires a STATE TAG`, this check should probably be removed.

```js
import { isValidForm } from 'cerebral-provider-forms/operators';
import ItemForm from '../../computes/ItemForm';
export default [
  isValidForm(ItemForm(props`id`)),
  {
    true: [],
    false: []
];
```

**Comment by henri-hulski** [comment]
I think this is also true for the other operators setField and resetForm.

**Comment by garth** [comment]
@henri-hulski I think that the other operators are a little different. They will update your state. If you form data is derived via a compute, then it will be very hard for setField and resetForm to know what do do.

**Comment by henri-hulski** [comment]
Yeah I didn't think about computed. But what happens when the form is passed as prop tag? Does that work now?

**Comment by garth** [comment]
it would work for isFormValid, any tag, compute (or literal, but why?) passed to form that resolves to a form value should work after #745.
